### PR TITLE
v17.3.1

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Thanks you for your contribution to the Azure-SDK-for-Go! We will triage and review it as quickly as we can.

As part of your submission, please make sure that you can make the following assertions:

 - [ ] I'm not making changes to Auto-Generated files which will just get erased next time there's a release.
   - If that's what you want to do, consider making a contribution here: https://github.com/Azure/autorest.go
 - [ ] I've tested my changes, adding unit tests where applicable.
 - [ ] I've added Apache 2.0 Headers to the top of any new source files.
 - [ ] I'm submitting this PR to the `dev` branch, or I'm fixing a bug that warrants its own release and I'm targeting the `master` branch.
 - [ ] If I'm targeting the `master` branch, I've also added a note to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-go/blob/master/README.md).
 - [ ] I've mentioned any relevant open issues in this PR, making clear the context for the contribution.

### CODE REVIEW

**Comment by joshgav** [comment]
@jhendrixMSFT can we pick up the latest containerservice spec too? Thanks!

**Comment by jhendrixMSFT** [comment]
@joshgav I'll do a full minor update early next week.

**Comment by joshgav** [comment]
Okay thanks. Can you pick up the updated README from https://github.com/Azure/azure-sdk-for-go/commit/b9f76dbde4ccc31ebee29b1a2974885a613f8f87?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## `v17.3.1`
+
+### Updated Services
+
+|          Package Name |     API Version     |
+| --------------------: | :-----------------: |
+|               network | 2017-10-01<br/>2017-11-01<br/>2018-01-01<br/>2018-02-01<br/>2018-04-01<br/>2018-05-01 |
+|         servicefabric | 5.6<br/>6.0<br/>6.1<br/>6.2 |
```
Comment (vladbarosan) [inline CHANGELOG.md]: do we want to specify what fix we did  ?

**Pull request merged**